mlflow/entities/trace.py

```diff
@@ -77,10 +77,14 @@ def _repr_mimebundle_(self, include=None, exclude=None):
         which contains a JSON representation of the Trace object. This object is deserialized
         in Databricks notebooks to display the Trace object in a nicer UI.
         """
-        return {
-            "application/databricks.mlflow.trace": self._serialize_for_mimebundle(),
-            "text/plain": repr(self),
-        }
+        from mlflow.tracing.display import get_display_handler
+
+        bundle = {"text/plain": repr(self)}
+
+        if not get_display_handler()._disabled:
```

Comment:
```suggestion
        if not get_display_handler().disabled:
```
nit: Since now we access the flag outside the class, let's make it a public but read-only property